doc/rados/configuration/mon-config-ref.rst

```diff
@@ -1145,32 +1147,32 @@ Miscellaneous
 :Default: ``False``
 
 
-``mon compact on bootstrap``
+``mon_compact_on_bootstrap``
 
 :Description: Compact the database used as Ceph Monitor store on
-              on bootstrap. Monitor starts probing each other for creating
-              a quorum after bootstrap. If it times out before joining the
-              quorum, it will start over and bootstrap itself again.
+              on bootstrap. Monitors probe each other to establish
```

Comment:
```suggestion
              bootstrap. Monitors probe each other to establish
```
"on" is redundant here (see the previous line in the text).

Reply:
ack ack